src/js/components/Menu/Menu.js

```diff
@@ -23,175 +24,226 @@ const ContainerBox = styled(Box)`
   ${props => props.theme.menu.extend};
 `;
 
-class Menu extends Component {
-  static defaultProps = {
-    dropAlign: { top: 'top', left: 'left' },
-    dropProps: {},
-    items: [],
-    messages: { openMenu: 'Open Menu', closeMenu: 'Close Menu' },
-    justifyContent: 'start',
-  };
+/* Notes on keyboard interactivity (based on W3C) // For details reference: https://www.w3.org/TR/wai-aria-practices/#menu
 
-  buttonRefs = {};
+To open menu when menu button is focused:
+- Space/Enter/Up arrow/Down arrow will open menu
 
-  constructor(props) {
-    super(props);
+To navigate within menu:
+- Up/down arrow keys can be used and will loop through options (keeping focus within the Menu)
+- Tab can be used, but once the last menu item is reached, Tab will close the Menu and continue through page content.
 
-    this.state = { activeItemIndex: -1, open: props.open || false };
-  }
+To close the menu:
+- Tabbing beyond the first or last menu item.
+- Esc will close the menu
+- Select a menu item
+
+To make a selection:
+- Enter key is pressed.
+- Space is pressed.
+*/
+
+const Menu = forwardRef((props, ref) => {
```

Comment:
We don't need `forwardRef` and `withForwardRef`, they do similar things. I think we should remove the `forwardRef` here.

Reply:
Gotcha. Removed it in recent commit.